src/renderers/WebGLRenderer.js

```diff
@@ -277,7 +277,7 @@ function WebGLRenderer( parameters ) {
 		morphtargets = new WebGLMorphtargets( _gl );
 		programCache = new WebGLPrograms( _this, extensions, capabilities );
 		renderLists = new WebGLRenderLists();
-		renderStates = new WebGLRenderStates();
+		renderStates = new WebGLRenderStates( properties );
```

Comment:
Why have you added this line? The ctor WebGLRenderStates has no arguments.

Reply:
This was indeed wrong